### ISSUE TITLE
Build instructions don't work

### ISSUE THREAD

**Issue by drder66** [gerbera/gerbera#766]
Just trying to build for the first time.
Git clone ok

cmake ../gerbera -DWITH_MAGIC=1 -DWITH_MYSQL=1 -DWITH_CURL=1 -DWITH_JS=1 -DWITH_TAGLIB=1 -DWITH_AVCODEC=1 -DWITH_FFMPEGTHUMBNAILER=1 -DWITH_EXIF=1 -DWITH_LASTFM=1

CMake Error: The source directory "../gerbera/gerbera" does not exist.

>Correct, there is no gerbera directory. There is gerbera-web, but that doesn't work

CMake Error: The source directory "./gerbera/gerbera-web" does not appear to contain CMakeLists.txt

>Correct, there is no CMakeLists.txt there. There is one in the root directory though. So on to next try:

cmake ../. -DWITH_MAGIC=1 -DWITH_MYSQL=1 -DWITH_CURL=1 -DWITH_JS=1 -DWITH_TAGLIB=1 -DWITH_AVCODEC=1 -DWITH_FFMPEGTHUMBNAILER=1 -DWITH_EXIF=1 -DWITH_LASTFM=1

>This looks like it might be working. Then:

By not providing "Findfmt.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "fmt", but
  CMake did not find one.

  Could not find a package configuration file provided by "fmt" with any of
  the following names:

    fmtConfig.cmake
    fmt-config.cmake

  Add the installation prefix of "fmt" to CMAKE_PREFIX_PATH or set "fmt_DIR"
  to a directory containing one of the above files.  If "fmt" provides a
  separate development package or SDK, be sure it has been installed.

>What next please?

**Comment by whyman** [comment]
Hi,
Generally with CMake you are expected to run the command from a folder next to the source code

> ```
> mkdir build
> cd build
> cmake ../gerbera

This is called an out of tree build. Its good because you can delete that directory and start clean very easily.

As for the `Findfmt` issue, that means that you do not have `libfmt` installed on your system - Admitelly we should probably provide a friendlier message that the generic CMake output.

Hope that helps!

**Comment by drder66** [comment]
I tried an email reply on this yesterday, but hadn't noted that didn't feed back here.

Yes, I followed the mkdir build etc. but I'm still puzzled that there's no gerbera directory in the build, only gerbera-web

After following the steps below, cmake stops with

CMake Error: The source directory "/home/der/gerbera/gerbera" does not exist

Which indeed it does not. /home/der/gerbera/gerbera/build does, /home/der/gerbera/gerbera-web does.

CMake ../../gerbera from gerbera/build does work, but it's not what the instructions say

Am I making any sense?
